mem/mem.h

```diff
@@ -73,6 +73,19 @@
 
 #ifndef MEM_H
 #define MEM_H
+#include <cstdlib>
+#include <type_traits>
+
+template<typename T> static inline T *mem_rmalloc()
+{
+	static_assert(std::is_trivially_constructible_v<T> && std::is_trivially_destructible_v<T>);
+	return static_cast<T *>(::malloc(sizeof(T)));
```

Comment:
I'm curious about this practice, why using `::malloc` rather than `std::malloc`?

Reply:
old bad habits 🥲